x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -797,7 +797,7 @@ observe the activity of the input.
 | `sqs_messages_inflight_gauge`             | Number of SQS messages inflight (gauge).
 | `sqs_messages_returned_total`             | Number of SQS message returned to queue (happens on errors implicitly after visibility timeout passes).
 | `sqs_messages_deleted_total`              | Number of SQS messages deleted.
-| `sqs_messages_waiting_gauge`              | Number of SQS messages waiting in the SQS queue (gauge). The value is refreshed every minute via data from GetQueueAttributes.
+| `sqs_messages_waiting_gauge`              | Number of SQS messages waiting in the SQS queue (gauge). The value is refreshed every minute via data from GetQueueAttributes. A value of (0) indicates `no messages waiting` but a value 0f (-1) indicates `metrics not collected/error collecting metrics `
```

Comment:
```suggestion
| `sqs_messages_waiting_gauge`              | Number of SQS messages waiting in the SQS queue (gauge). The value is refreshed every minute via data from GetQueueAttributes. A value of (0) indicates `no message waiting` but a value of (-1) indicates `metrics not collected/error collecting metrics `
```

Reply:
Updated to https://github.com/elastic/beats/pull/36101#discussion_r1268545974